website/static/js/addonSettings.js

```diff
@@ -96,7 +96,7 @@ var OAuthAddonSettingsViewModel = oop.defclass({
                     } else if (self.name === 'bitbucket') {
                         self.setMessage('Add-on successfully authorized. If you wish to link a different account, log out of bitbucket.org before attempting to connect to a second Bitbucket account on the OSF. This will clear the credentials stored in your browser.', 'text-success');
                     } else if (self.name === 'onedrive') {
-                        self.setMessage('Add-on successfully authorized. If you wish to link a different account, log out of onedrive.live.com before attempting to connect to a second OneDrive account on the OSF. This will clear the credentials stored in your browser.', 'text-success');
+                        self.setMessage('Add-on successfully authorized. If you wish to link a different account, log out of Personal OneDrive at login.live.com and OneDrive for School or Business at login.microsoftonline.com before attempting to connect to a second OneDrive account on the OSF. This will clear the credentials stored in your browser.', 'text-success');
```

Comment:
Comment, I was thinking that we can use a dedicated logout URL but then I realized it was a bad idea. Go to the app and click the sign-out button is convenient and safe. The link won't change user's Microsoft account state when it is clicked or previewed.

Reply:
Q: was this a suggestion or a confirmation?  I'm not crazy about the wording as it is now, but I couldn't figure out a way to logout out of both personal AND 4B accounts more succinctly.